exawind/Pazy_Wing/README.md

```diff
@@ -0,0 +1,177 @@
+
+   ```bash
+   # Case selection
+   aoa=93
+   windspeed=30
+   # Simulation parameters, experts only
+   dtshort=0.00002
+   dtlong=0.00006
+   density=1.225
+   visc=0.000018
+   tkein=1.0
+   sdrin=250.0
+   amrnx=128
+   cfdplotinterval=1000
+   cfdcheckpointinterval=10000
+   ```
+
+   Variable description:
+   - `aoa` - The angle of attack, given in degrees.  Since the nominal position of the wing is with the leading edge perpindicular to the incoming flow (per OpenFAST conventions), `aoa` should be set 90 degrees higher than the true angle of attack.  The angles of attack used in the experiment are 3, 5, and 7 degrees, so the corresponding values of `aoa` are 93, 95, and 97 degrees.
+   - `windspeed` - The freestream wind velocity, given in meters per second.
+   - `dtshort` - The structural timestep, in seconds.
+   - `dtlong` - The CFD and driver timestep, in seconds.  This should be an integer multiple of `dtshort`.
+   - `density` - The freestream air density, in kg/m^3.
+   - `visc` - The freestream air dynamic viscosity, in Pa-s.
+   - `tkein` and `sdrin` - The freestream turbulence model parameters.
+   - `amrnx` - The number of elements in the x and y directions of the base AMR-Wind grid.
+   - `cfdplotinterval` - The number of timesteps that pass in between plot file output.
+   - `cfdcheckpointinterval` - The number of timesteps that pass in between checkpoint file output.
+
+   If the user wishes to reproduce the benchmark results, only `aoa` and `windspeed` need to be modified.
+
+7. Run the `setup.sh` script.  This will copy the simulation input files from `base` and substitute in the simulation parameters you added to the header using `aprepro`.  The resulting files will be placed in a directory titled `a#v#`, where the first number indicates the value of `aoa`, and the second indicates the value of `windspeed`.  This naming convention is chosen to facilitate parameter sweeps.
+
+8. Navigate to the simulation directory you just created, and submit the slurm script.
+
+**Code Versions**
+
+While we expect that using the current release of each code will produce comparable results, the specific versions of each code that were used to generate the results in this document are given below for completeness.
+
+- ExaWind driver: [``046b080c7e0f0ab1efea0dccb4f798fe84ef905e``]( https://github.com/Exawind/exawind-driver/commit/046b080c7e0f0ab1efea0dccb4f798fe84ef905e)
+- Nalu-Wind:  &ensp;&ensp;&nbsp;&nbsp;&nbsp;  [``b9e4ae654b646ecd0501dd6391dc7537239c82db``](https://github.com/Exawind/nalu-wind/commit/b9e4ae654b646ecd0501dd6391dc7537239c82db)
+- AMR-Wind:   &ensp;&emsp;&nbsp; [``b61c01895e8eab388e4a3fd129e1db0f4fd0f534``]( https://github.com/Exawind/amr-wind/commit/b61c01895e8eab388e4a3fd129e1db0f4fd0f534) 
+- OpenFAST: &ensp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[``024dbc1816ca8caeefcc720b1099397730b1ec0a``](https://github.com/Exawind/amr-wind/commit/024dbc1816ca8caeefcc720b1099397730b1ec0a)
```

Comment:
The OpenFAST link doesn't work, because I think it points to the AMR-Wind repo.  Do we know if this is openfast v3.5 or v4?

Reply:
Oops!  Good catch.  Yeah, the SHA is right, I just copy-pasted the first part, and didn't verify it worked.
I fixed the link.

This is 3.5.  I've been working on re-running everything with 4 to make sure nothing changes.